react-backend/client/src/components/home.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const postFeedback = (cbId, feedbackScore) => {
+  const headers = new Headers({
+    Authorization: localStorage.getItem('token'),
+    'Content-Type': 'application/json',
+  });
+
+  return fetch('/api/cb/feedback', {
+    method: 'POST',
+    headers,
+    body: JSON.stringify({
+      cbId,
+      feedbackScore,
+    }),
+  });
+};
 const Home = () => (
   <div>
     <h1>
       Who are you?<br />
     </h1>
+    <button onClick={() => postFeedback(1, -1)}>-1</button>
```

Comment:
where are these id's coming from?

Reply:
Yup, that explains the overwhelming feeling that I forgot something 😊